client/state/jetpack-connect/test/actions.js

```diff
@@ -6,6 +6,7 @@
  * Internal dependencies
  */
 import * as actions from '../actions';
+import * as loginActions from 'state/login/actions';
```

Comment:
Why not import them individually?

Reply:
So I can `jest.spyOn( object, methodName )` easily to mock them.